Style-Guide/Code-Layout-and-Formatting.md

```diff
@@ -166,30 +197,36 @@ $variable=Get-Content $FilePath -Wai:($ReadCount-gt0) -First($ReadCount*5)
 $variable = Get-Content -Path $FilePath -Wait:($ReadCount -gt 0) -TotalCount ($ReadCount * 5)
 ```
 
-#### Spaces around special characters
+### Spaces around special characters
 
-White-space is (mostly) irrelevant to PowerShell, but its proper use is the key to writing easily readable code.
+White-space is (mostly) irrelevant to PowerShell, but its proper use is key to writing easily readable code.
 
 Use a single space after commas and semicolons, and around pairs of curly braces.
 
-Avoid extra spaces inside parenthesis or square braces.
+Avoid unnecessary extra spaces inside parenthesis or square braces.
 
-Nested expressions `$( ... )` and script blocks `{ ... }` should have a single space _inside_ them to make code stand out and be more readable.
+Nested expressions `$( ... )` and script blocks `{ ... }` should have a single space _inside_ the enclosing braces or parentheses to make code stand out and be more readable.
 
-Nested expressions `$( ... )` and variable delimiters `${...}` inside strings do not need spaces _outside_, since that would become a part of the string.
+Nested expressions `$( ... )` and variable delimiters `${...}` nested inside strings should not include additional space _surrounding_ them, unless it is desired for the final string to include them.
 
+```powershell
+$Var = 'out'
+"This is a string with${Var} unnecessary spaces."
```

Comment:
This example is **way** too clever -- the reader has to parse it for the sentence to even make sense, and until they do, it's not at all clear what it's meant to be an example of.

Reply:
Mm. I feel as though there ought to be some kind of example here, but tbh I'm a little grasping at straws for what, exactly, makes sense here. I'll think on it.